pkg/schemas/management.cattle.io/v3/schema.go

```diff
@@ -618,7 +618,43 @@ func authnTypes(schemas *types.Schemas) *types.Schemas {
 			schema.ResourceMethods = []string{http.MethodGet, http.MethodPut}
 		}).
 		MustImport(&Version, v3.GoogleOauthConfigApplyInput{}).
-		MustImport(&Version, v3.GoogleOauthConfigTestOutput{})
+		MustImport(&Version, v3.GoogleOauthConfigTestOutput{}).
+		//OIDC Config
+		MustImportAndCustomize(&Version, v3.OIDCConfig{}, func(schema *types.Schema) {
+			schema.BaseType = "authConfig"
+			schema.ResourceActions = map[string]types.Action{
+				"disable": {},
+				"configureTest": {
+					Input:  "oidcConfig",
+					Output: "oidcTestOutput",
+				},
+				"testAndApply": {
+					Input: "oidcApplyInput",
+				},
+			}
+			schema.CollectionMethods = []string{}
+			schema.ResourceMethods = []string{http.MethodGet, http.MethodPut}
+		}).
+		MustImport(&Version, v3.OIDCApplyInput{}).
+		MustImport(&Version, v3.OIDCTestOutput{}).
+		//KeyCloakOIDC Config
+		MustImportAndCustomize(&Version, v3.KeyCloakOIDCConfig{}, func(schema *types.Schema) {
+			schema.BaseType = "authConfig"
+			schema.ResourceActions = map[string]types.Action{
+				"disable": {},
+				"configureTest": {
+					Input:  "keycloakoidcConfig",
```

Comment:
Confused about casing here, why is it all lowercase but config ?

Reply:
changed the casing - there was no specific need for the casing to be all lower